src/x86/runtime.rs

```diff
@@ -0,0 +1,120 @@
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+#[macro_export]
+macro_rules! __unstable_detect_feature_x86 {
+    ("sse") => { ::stdsimd::vendor::__unstable_detect_feature(::stdsimd::vector::__Feature::sse{}) };
```

Comment:
I think the `::stdsimd` here may be better done as `$crate` (weird macro thing...)